packages/runner/src/Runner.ts

```diff
@@ -41,18 +55,57 @@
  *
  * myGame.update.emit(time);
  * ```
+ *
+ * Type safety:
+ *
+ * ```ts
+ *
+ * let runner: Runner<'update', [number]>;
```

Comment:
Minor suggestion for example code, otherwise this is a great example.

```suggestion
 * import { Runner } from '@pixi/runner';
 *
 * const runner: Runner<'update', [number]>;
```